<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDialog</class>
 <widget class="QDialog" name="SetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters </string>
  </property>
  <widget class="QPushButton" name="SetButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cannyCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Canny</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cannyThresh1Spin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Canny Threshold 1</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cannyThresh2Spin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Canny Threshold 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="linearTransCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Transform</string>
   </property>
  </widget>
  <widget class="QSlider" name="alphaSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Beta</string>
   </property>
  </widget>
  <widget class="QSlider" name="betaSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="grayCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB 2 Gray</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="gammaCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gamma correction</string>
   </property>
  </widget>
  <widget class="QSlider" name="gammaSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="hist1CheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram equalization computed</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hist2CheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram equalization function</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
